File: src/main/java/no/novari/flyt/authorization/client/sourceapplications/model/FSkyssSourceApplication.java
package no.novari.flyt.authorization.client.sourceapplications.model;

import org.springframework.beans.factory.annotation.Value;
import org.springframework.stereotype.Component;

@Component
public class FSkyssSourceApplication extends BaseSourceApplication {
    public FSkyssSourceApplication(
            @Value("${fint.flyt.fskyss.sso.client-id:#{null}}") String clientId,
            @Value("${fint.flyt.fskyss.available:false}") boolean available
    ) {
        super(8L, "FSkyss", clientId, available);
    }
}

File: src/main/kotlin/no/novari/flyt/authorization/client/sourceapplications/model/FSkyssSourceApplication.kt
package no.novari.flyt.authorization.client.sourceapplications.model

import org.springframework.beans.factory.annotation.Value
import org.springframework.stereotype.Component

@Component
class FSkyssSourceApplication(
    @Value("\${fint.flyt.fskyss.sso.client-id:#{null}}")
    clientId: String?,
    @Value("\${fint.flyt.fskyss.available:false}")
    available: Boolean,
) : BaseSourceApplication(
        id = 8L,
        displayName = "FSkyss",
        clientId = clientId,
        available = available,
    )
